src/core_plugins/metrics/public/components/aggs/percentile.js

```diff
@@ -43,37 +44,46 @@ class Percentiles extends Component {
     if (model.mode === 'line') {
       optionsStyle.display = 'none';
     }
+    const htmlId = htmlIdGenerator(model.id);
     return  (
       <div className="vis_editor__percentiles-row" key={model.id}>
         <div className="vis_editor__percentiles-content">
           <input
+            aria-label="Percentile"
```

Comment:
I just confirmed with @aphelionz that we shouldn't use `aria-label` on `input` elements. Instead, we should be using for/id attributes to associate inputs with `label` elements, and `aria-describedby` to associate them with help an error text per https://github.com/elastic/kibana/blob/master/style_guides/accessibility_guide.md#label-every-form-element.

CC @timroes We should probably re-assess accessibility in TSVB. I'm going to reopen https://github.com/elastic/kibana/issues/12902 and list this issue and some others I've found.